cfgov/templates/wagtailadmin/login.html

```diff
@@ -25,9 +25,9 @@ <h1>Log in to consumerfinance.gov</h1>
             {% block login_form %}
             {% csrf_token %}
 
-            {% url 'user_welcome' as home_url %}
```

Comment:
I removed this because we have no `user_welcome` URL and so this was always null, and thus never used.